platform.in

```diff
@@ -66,10 +66,21 @@
 %__arch_install_post   @ARCH_INSTALL_POST@
 
 %__os_install_post    \
-    %{_rpmconfigdir}/brp-compress \
+    %{?!dont_compress:			[ -n "$DONT_COMPRESS" ]      	|| %{_rpmconfigdir}/brp-compress} \
     %{_rpmconfigdir}/brp-strip %{__strip} \
     %{_rpmconfigdir}/brp-strip-static-archive %{__strip} \
     %{_rpmconfigdir}/brp-strip-comment-note %{__strip} %{__objdump} \
+    %{?!dont_clean_files:		[ -n "$DONT_CLEANUP" ]       	|| %{_rpmconfigdir}/brp-clean-files} \
+    %{?!dont_remove_info_dir:		[ -n "$DONT_REMOVE_INFO_DIR" ]	|| %{_rpmconfigdir}/brp-remove-info-dir} \
+    %{?!dont_relink:			[ -n "$DONT_RELINK" ]        	|| %{_rpmconfigdir}/brp-relink-symlinks} \
+    %{?!dont_remove_libtool_files:	[ -n "$DONT_REMOVE_LIBTOOL_FILES" ] || %{_rpmconfigdir}/brp-remove-libtool-files} \
+    %{?!dont_fix_eol:			[ -n "$DONT_FIX_EOL" ]		|| %{_rpmconfigdir}/brp-fix-eol} \
+    %{?!dont_remove_rpath:		[ -n "$DONT_REMOVE_RPATH" ]	|| %{_rpmconfigdir}/brp-remove-rpath} \
+    %{?!dont_fix_file_permissions: 	[ -n "$DONT_FIX_FILE_PERMISSIONS" ] || %{_rpmconfigdir}/brp-fix-file-permissions} \
+    %{?!dont_fix_xdg:			[ -n "$DONT_FIX_XDG" ]       	|| %{_rpmconfigdir}/brp-fix-xdg} \
```

Comment:
This should be renamed to something more obvious. Maybe `fix_desktop_files`. The name "XDG" isn't very helpful...